docs/internals/deprecation.txt

```diff
@@ -35,6 +35,10 @@ details on these changes.
 * The ``map_width`` and ``map_height`` attributes of ``BaseGeometryWidget``
   will be removed.
 
+* The deprecated spelling ``assertFormsetError()`` will be removed.
+
+* The deprecated spelling ``assertQuerysetEqual()`` will be removed.
```

Comment:
All points in this page are about previously deprecated features, so `deprecated` is unnecessary.
```suggestion
* The ``SimpleTestCase.assertFormsetError()`` method will be removed.

* The ``TransactionTestCase.assertQuerysetEqual()`` method will be removed.
```